paddle/framework/lod_tensor.h

```diff
@@ -21,7 +21,6 @@
 #include <thrust/system/cuda/experimental/pinned_allocator.h>
 #endif
 
-#include <glog/logging.h>
```

Comment:
Thanks for the very detail-oriented work style -- moving the inclusion of glog header file into the .cu file.